app/src/main/res/values-pt-rBR/strings.xml

```diff
@@ -0,0 +1,478 @@
+<resources>
+    <string name="app_name" translatable="false">FastHub</string>
+    <string name="app_name_full" translatable="false">FastHub for Android</string>
```

Comment:
hey there, what should I do with the translatable="false" strings? delete them from my file or should I translate them?